### DISCUSSION TITLE
Swap TUS to AVAX

### DISCUSSION THREAD

**Discussion by isidororeyes** [coccoinomane/crabada.py#97] [Q&A] [answered]
Hello!

I'm trying to automate the mining and now I need to manually send TUS to AVAX for gas fees.
I've created an alert to notify when I'm below a certain threshold, and now I'm trying to use the buildTransactionWithValue but I'm lost.

Can anyone give me some pointers on how to transfer TUS to AVAX using the crabada.py framework?
Thanks!!

**Comment by coccoinomane** [comment, marked as answer]
Hi @isidororeyes !

An automatic swap feature was implemented by @EugeneTeu , but eventually I decided not to include it in crabada.py, the reason being that crabada.py is not a framework but a bot.

Nonetheless, @EugeneTeu was very nice and left the PR available to everybody, you can find it here > https://github.com/coccoinomane/crabada.py/pull/80.

Cheers,
Cocco
